### PR TITLE
Optimize yq installation

### PR DESCRIPTION
**Pull request by liangxia**
Use an image which already contains yq, so that we are no longer relying on curl, which is not stable enough.

### CODE REVIEW

**Comment by liangxia** [comment]
Waiting for https://github.com/openshift/release/pull/31245

File: ci-operator/step-registry/sdn/conf/sdn-conf-ref.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ref:
   as: sdn-conf
   from_image:
-   namespace: origin
-   name: centos
-   tag: '8'
+    namespace: ocp
+    name: cli-jq
```
Comment (jianlinliu) [inline ci-operator/step-registry/sdn/conf/sdn-conf-ref.yaml:5]: `cli-yq`?

Comment (jianlinliu) [inline ci-operator/step-registry/sdn/conf/sdn-conf-ref.yaml:5]: `cli-yq`?

**Comment by liangxia** [comment]
The image `cli-yq` does not get build automatically, dptp help build it manually.

/test all

**Comment by liangxia** [comment]
In the failed log which happens on last Friday,
```
ERRO[2022-08-12T01:21:53Z] 
  * could not resolve inputs: could not determine inputs for step [input:ocp-cli-yq-latest]: could not resolve base image from ocp/cli-yq:latest: imagestreamtags.image.openshift.io "cli-yq" not found 
```
In the latest log,
```
INFO[2022-08-17T01:12:45Z] Tagging ocp/cli-yq:latest into pipeline:ocp-cli-yq-latest. 
...
INFO[2022-08-17T01:15:58Z] Running step installer-rehearse-aws-sdn-conf. 
INFO[2022-08-17T01:16:18Z] Step installer-rehearse-aws-sdn-conf succeeded after 20s. 
```

In short, PR ready for review.

**Comment by liangxia** [comment]
In favor of https://github.com/openshift/release/pull/31405

/close

**Pull request closed without merging**